test/run

```diff
@@ -15,4 +15,5 @@ if [ -n "$TEST_SCENARIO" ]; then
 else
   test/check-cli TestImages
   test/check-api
+  test/check-lorax
```

Comment:
How long does this script take ? From what I can tell it builds a boot.iso so you may want to add a new context/TEST_SCENARIO for it instead.